config/emay.json

```diff
@@ -0,0 +1,25 @@
+{
+  "agoraUrlRoot": "https://agora.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "bardRoot": "https://terra-bard-dev.appspot.com",
+  "bondUrlRoot": "https://bond.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "calhounUrlRoot": "https://calhoun.dsde-dev.broadinstitute.org",
+  "catalogUrlRoot": "https://catalog.dsde-dev.broadinstitute.org",
+  "dataRepoUrlRoot": "https://data.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "devUrlRoot": "https://bvdp-saturn-dev.appspot.com",
+  "dockstoreUrlRoot": "https://staging.dockstore.org",
+  "externalCredsUrlRoot": "https://externalcreds.dsde-dev.broadinstitute.org",
+  "firecloudBucketRoot": "https://storage.googleapis.com/firecloud-alerts-dev",
+  "firecloudUrlRoot": "https://firecloud-fiab.dsde-dev.broadinstitute.org",
+  "jobManagerUrlRoot": "https://job-manager.fiab-emay-moving-deer.bee.envs-terra.bio/jobs",
+  "leoUrlRoot": "https://leonardo.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "marthaUrlRoot": "https://martha.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "orchestrationUrlRoot": "https://firecloudorch.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "rawlsUrlRoot": "https://rawls.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "rexUrlRoot": "https://terra-rex-dev.appspot.com",
+  "samUrlRoot": "https://sam.fiab-emay-moving-deer.bee.envs-terra.bio",
+  "shibbolethUrlRoot": "https://broad-shibboleth-prod.appspot.com/dev",
+  "alphaSpendReportGroup": "Alpha_Spend_Report_Users",
+  "alphaRegionalityGroup": "Alpha_Regionality_Users",
+  "terraDockerVersionsFile": "terra-docker-versions-dev.json",
+  "terraDockerImageBucket": "terra-docker-image-documentation-dev"
```

Comment:
Should `emay` references in `UrlRoot`s be replaced?

Reply:
See above comment.